packages/react-i18n/src/constants/index.ts

```diff
@@ -28,7 +28,7 @@ export const dateStyle = {
   },
 };
 
-export enum Weekdays {
+export enum Weekday {
```

Comment:
I think this will technically be a breaking change (probably fine not to treat it as breaking since we just did 1.0 and we can sneak this in as a patch to that :P

Reply:
But make sure to reference it on the changelog